### PR TITLE
Should GetVirtualPath returns PathString

### PR DESCRIPTION
**Pull request by ianhong**
@rynowak 

Please use this PR for this issue, #31. Ignore the previous one.

Thanks

### CODE REVIEW

**Comment by ianhong** [comment]
@rynowak 

By the way, I didn't change any thing for the product codes from the previous [Design] PR. I just updated the tests.

File: test/Microsoft.AspNet.Routing.Tests/RouteCollectionTest.cs [hunk 1]
```diff
@@ -45,15 +45,15 @@ public class RouteCollectionTest
             var pathData = routeCollection.GetVirtualPath(virtualPathContext);
 
             // Assert
-            Assert.Equal(lowercaseUrl, pathData.VirtualPath);
+            //Assert.Equal(lowercaseUrl, pathData.VirtualPath);
```
Comment (rynowak) [inline test/Microsoft.AspNet.Routing.Tests/RouteCollectionTest.cs]: why is this cmmenting out?

**Comment by rynowak** [comment]
:shipit: after uncommenting, Need to see an MVC reaction PR as well.

**Comment by ianhong** [comment]
ff209f04bb2daded5ae49fae015b6631d8a861d0

**Pull request closed without merging**